Source/WebCore/loader/HistoryController.cpp

```diff
@@ -227,7 +227,8 @@ void HistoryController::saveDocumentAndScrollState()
             continue;
         CheckedRef history = localFrame->history();
         history->saveDocumentState();
-        history->saveScrollPositionAndViewStateToItem(history->currentItem());
+        RefPtr currentItem = history->currentItem();
+        history->saveScrollPositionAndViewStateToItem(currentItem.get());
```

Comment:
Should use `history()->protectedCurrentItem().get()`

Reply:
Sure, fixing.